### PR TITLE
Add temporary script to update keycloak user group.

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: update-keycloak-user-group/src/main/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+keycloak {
+  url = "https://auth.tdr-"${?ENVIRONMENT}".nationalarchives.gov.uk"
```
Comment (TomJKing) [inline update-keycloak-user-group/src/main/resources/application.conf:2]: Will that work for prod? The url is: `auth.tdr.national.gov.uk for prod`.

File: update-keycloak-user-group/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Update Keycloak User Group
+This script is used to add users to the judgment group in Keycloak.
+
+## Running the script
+There are two environment variables
+* `ENVIRONMENT` - This is used to build the url for Keycloak so needs to be either, integration, staging or empty string.
```
Comment (TomJKing) [inline update-keycloak-user-group/README.md:6]: What about the prod environment?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments.

File: update-keycloak-user-group/src/main/resources/application.conf (hunk 1 above)
Comment (MancunianSam) [reply update-keycloak-user-group/src/main/resources/application.conf:2]: No, it won't, missed that. I've changed the environment variable to be SUB_DOMAIN so you can set it as tdr-integration or tdr.

File: update-keycloak-user-group/README.md (hunk 2 above)
Comment (MancunianSam) [reply update-keycloak-user-group/README.md:6]: I've updated this to say you need to set SUB_DOMAIN instead of ENVIRONMENT.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks for doing this at such short notice.

**Pull request merged**